packages/features/sales-escrows/modals/StartEscrowModal.tsx

```diff
@@ -127,16 +142,20 @@ export function StartEscrowModal({
   const validateForm = () => {
     let errors = { amount: '', token: undefined };
     const fee = toLargerUnit(token.fee, token.decimals);
+    const amount = toBigNumber(enteredAmount);
```

Comment:
Please move this line back to where it was below line 153 or it will throw an error trying to convert a potentially invalid input value.